CMakeLists.txt

```diff
@@ -139,7 +139,7 @@ IF (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
 
     ENDIF ()
 
-    SET (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wpedantic -Wshadow -Wno-deprecated -Wl,--no-undefined")
+    SET (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wpedantic -Wshadow -Wno-deprecated")
```

Comment:
Is this still required?

Reply:
Adding that C++ compiler flag triggers some `undefined references` related to pybind11 and python. My understanding is that this is related to the linking stage and python headers but no solution on that point for now